### PR TITLE
UEBAEssentials_ContentTesting1

### PR DESCRIPTION
**Pull request by vakohl**
Required items, please complete
   
   Change(s):
   - Case-sensitivity, Entity Mapping, Description updates

   Reason for Change(s):
   - Rules re-validation

   Version Updated:
   - NA

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

File: Solutions/UEBA Essentials/Hunting Queries/Anomalous Activity Role Assignment.yaml [hunk 1]
```diff
@@ -15,15 +15,20 @@ relevantTechniques:
 query: |
     let operations = dynamic(['Create role assignment']);
     BehaviorAnalytics
-    | where ActionType in(operations)
+    | where ActionType in~ (operations)
     | where ActivityInsights contains "True"
```
Comment (rahul0216) [inline Solutions/UEBA Essentials/Hunting Queries/Anomalous Activity Role Assignment.yaml:19]: You can use 'has' instead of 'contains'.

**Review by rahul0216 — COMMENTED** [review]
1. Please address one inline comment.
2. You can add version also in yaml files.

File: Solutions/UEBA Essentials/Hunting Queries/Anomalous Activity Role Assignment.yaml (hunk 1 above)
Comment (vakohl) [reply Solutions/UEBA Essentials/Hunting Queries/Anomalous Activity Role Assignment.yaml]: Thankyou @rahul0216 . This is done

**Review by rahul0216 — APPROVED** [review]
Changes look good.

**Pull request merged**